seed/static/seed/js/services/analyses_service.js

```diff
@@ -197,6 +197,12 @@ angular.module('BE.seed.service.analyses', [])
                           })
                       }
                     })
+                    .catch(data => {
+                      // yikes, something went wrong. Let the caller know the status
+                      // probably changed and let's bail
+                      status_update_callback(id)
+                        .then(() => no_current_task_callback(id))
+                    })
```

Comment:
handle the case where we fail the analysis (and as a result the progress data) -- the called method will reject the promise which we weren't doing anything with before.